docs-starlight/src/content/docs/03-features/07-scaffold.md

```diff
@@ -75,8 +75,8 @@ If you define input variables in your boilerplate template, Terragrunt will prom
 There are also a set of variables that Terragrunt will automatically expose to your boilerplate templates for rendering:
 
 - `sourceUrl` - URL to module
-- `requiredVariables` - list of required variables in the module being scaffolded (see below)
-- `optionalVariables` - list of optional variables in the module being scaffolded (see below)
+- `requiredVariables` - list of required variables in the unit being scaffolded (see below)
```

Comment:
The module is what's being scaffolded _as_ a unit, as I understand it.

Reply:
I was thinking about it, and my conclusion was we're _instantiating_ a module by writing new code (a unit), and we're _scaffolding_ that new unit code, not hte module code. Thoughts?

Reply:
Works for me.